app/controllers/api_controller/manager.rb

```diff
@@ -87,7 +87,9 @@ def target_resource_method(is_subcollection, type, action)
       else
         target = "#{action}_resource"
         typed_target = "#{target}_#{type}"
-        respond_to?(typed_target) ? typed_target : target
+        return typed_target if respond_to?(typed_target)
+        return target if respond_to?(target)
+        return "custom_action_resource" if resource_can_have_custom_actions(type)
```

Comment:
This looks like it can now return `nil` where it couldn't before. Is that intentional?